.prow.yaml

```diff
@@ -0,0 +1,31 @@
+presubmits:
+  - name: kustomize-build
+    decorate: true
+    max_concurrency: 1
+    run_if_changed: ".*yaml"
+    skip_report: false
+    context: aicoe-ci/prow/kustomize-build
+    spec:
+      containers:
+        - image: quay.io/operate-first/opf-toolbox:v0.3.1
+          command:
+            - "test-kustomize-build"
```

Comment:
is this to run from the image? if so, isn't the file kept at /usr/local/bin/ ?

Reply:
resolved in `v0.3.2` - can be called directly